### PR TITLE
adding refresh scripts

### PR DESCRIPTION
**Pull request by yfaktor**
adding a simple refresh Oauth token script in JS and Python

### CODE REVIEW

**Comment by yfaktor** [comment]
@ksvirkou-hubspot @asalauyeu-hubspot

**Comment by asalauyeu-hubspot** [comment]
Should these scripts be presented in the root folder? If we want to have an ability to download them via hubspot/cli, we need to add them to appropriate language folders, since `hs create` downloads only language folder, not whole project.

**Comment by yfaktor** [comment]
> Should these scripts be presented in the root folder? If we want to have an ability to download them via hubspot/cli, we need to add them to appropriate language folders, since `hs create` downloads only language folder, not whole project.

so what do you suggest for the path here?

**Comment by yfaktor** [comment]
@asalauyeu-hubspot pl see if this makes more sense now

File: node/code-snippets/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# HubSpot refresh token JavaScripr script
```
Comment (asalauyeu-hubspot) [inline node/code-snippets/README.md:1]: If we add to code-snippets folder general README, seems its description should be generalized too, so we're able to add more snippets here in the future if necessary. For example title
"Hubspot code snippets scripts for using them *somewhere*"
and after that subtitle about refresh token script. Works also for python.
Also, please, fix the typo to **Javascript** not Javascripr

**Comment by asalauyeu-hubspot** [comment]
> @asalauyeu-hubspot pl see if this makes more sense now

Current structure looks good to me, but seems it requires more info to READMEs

**Pull request closed without merging**